.github/workflows/testing.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v5
```

Comment:
```suggestion
        uses: actions/checkout@v4
```

Reply:
thx! I accidentally changed the wrong action, I wanted to change the setup-python action version

Reply:
> thx! I accidentally changed the wrong action, I wanted to change the setup-python action version

Yep! I think there's still an issue though. It seems like `3.7` is just not available anymore.